app-text/master-pdf-editor/master-pdf-editor-5.1.36.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils desktop
+
+DESCRIPTION="A complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+COMMON_DEPEND="
+	>=media-gfx/sane-backends-1.0
+	>=dev-qt/qtsvg-5.4.1:5
+	>=dev-qt/qtnetwork-5.4.1:5
+	>=dev-qt/qtgui-5.4.1:5
+	>=dev-qt/qtprintsupport-5.4.1:5
+"
+
+RDEPEND="${COMMON_DEPEND}"
+
+QA_PREBUILT="/opt/${PN}/masterpdfeditor5"
+
+S="${WORKDIR}/${PN}-${PV%%.*}"
+
+src_install() {
+	insinto /opt/${PN}
+	doins -r fonts lang stamps templates masterpdfeditor5.png
+
+	exeinto /opt/${PN}
+	doexe masterpdfeditor5
+
+	dosym ../${PN}/masterpdfeditor5 /opt/bin/masterpdfeditor5
```

Comment:
Style nitpick: I would move this line up one to keep it next to the thing it points to. Basically, keep related things together. (_In this paragraph, I'm talking about the executable...._)

Reply:
Thank you for your feedback. I made the suggested changes.